hydroshare/settings.py

```diff
@@ -178,11 +178,6 @@ def _(s): return s # noqa
 # Alternative tmp folder
 FILE_UPLOAD_TEMP_DIR = "/hs_tmp"
 
-# Sitemap for robots
-ROBOTS_SITEMAP_URLS = [
-    "http://localhost:8000/sitemap/",
-]
-
```

Comment:
removed because we don't want to reference "localhost" in other environments.
We could set this in `local_settings.py` for all of the environments but we don't have to.
Django Robots will correctly generate the sitemap reference in the robots.txt on its own:
https://django-robots.readthedocs.io/en/latest/#sitemaps